test/test_cuda.py

```diff
@@ -274,6 +275,7 @@ def assert_change(comp=1, empty_cache=False, reset_peak=False):
         assert_change(0, empty_cache=True)
         assert_change(0, reset_peak=True)
 
+    @unittest.skipIf(IS_JETSON, "Does not work on jetson")
```

Comment:
Do you know what errors are raised on Jetsons for this test?

Reply:
```
FAIL: test_cudart_register (__main__.TestCuda)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/nvidia/.local/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py", line 1470, in wrapper
    method(*args, **kwargs)
  File "/home/nvidia/.local/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py", line 1470, in wrapper
    method(*args, **kwargs)
  File "test_cuda.py", line 284, in test_cudart_register
    self.assertEqual(r, 0)
  File "/home/nvidia/.local/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py", line 2074, in assertEqual
    super().assertEqual(x, y, msg=msg)
AssertionError: <cudaError.???: 801> != 0

----------------------------------------------------------------------
Ran 168 tests in 42.730s
```